go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/fatih/color v1.8.0
 	github.com/filecoin-project/filecoin-ffi v0.30.4-0.20200716204036-cddc56607e1d
 	github.com/filecoin-project/go-address v0.0.4
-	github.com/filecoin-project/go-bitfield v0.2.1-0.20200920172649-837cbe6a1ed3
+	github.com/filecoin-project/go-bitfield v0.2.0
```

Comment:
@Kubuxu note: this _doesn't_ include the buffer pool change, in case you still wanted that.

Reply:
ahh, I used latest instead of an explicit tag, and go proxy didn't pick it up yet